sysl2/sysl/importer/writer.go

```diff
@@ -252,6 +254,9 @@ func (w *writer) writeDefinitions(types TypeList) {
 		switch {
 		case isBuiltInType(t):
 			// do nothing
+		case isUnionType(t):
+			w.writeLines(BlankLine)
+			w.writeUnion(t)
```

Comment:
This switch has become a very confused mix of type checks which looks like it could be done as a simple type-assertion switch. An added advantage would be that the switch variable would be of type `*Union`.

Reply:
Disagree, those is*Type() funcs are slightly coomplicated and the ordering matters, would prefer to leave as is